cfn-resources/integration-tests/project/project_template.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
[nit] It would be interesting to use [go templates](https://pkg.go.dev/text/template) here instead, so that parameters could be defined within the e2e test and automatically filled out.

It would remove the issue of needing to hard-code values such as OrgId